_data/projects.yml

```diff
@@ -133,3 +133,27 @@
   docs: "http://fiasco.readthedocs.io/en/latest/"
   code_url: "https://github.com/wtbarnes/fiasco"
   contact: "Will Barnes"
+
+- name: "OCBpy"
+  html_url: "https://github.com/aburrell/ocbpy"
+  description: "A Python module that converts between AACGM coordinates and an adjustable magnetic coordinate system based on the location of the polar cap"
+  logo: ""
+  docs: "http://ocbpy.readthedocs.io/en/latest/"
+  code_url: "https://github.com/aburrell/ocbpy"
+  contact: "Angeline G. Burrell"
+  
+- name: "AACGMV2"
+  html_url: "https://github.com/aburrell/aacgmv2"
+  description: "A Python wrapper for the AACGM-v2 C library"
+  logo: ""
+  docs: "http://aacgmv2.readthedocs.io/en/latest/"
+  code_url: "https://github.com/aburrell/aacgmv2"
+  contact: "Angeline G. Burrell"
+  
+- name: "AACGMV2"
```

Comment:
Is this supposed to be ApexPy?

Reply:
No, this is another magnetic coordinate system that is better suited to high latitudes.  Actually a more popular module than ApexPy.